S/SuiteSparse/ParU/build_tarballs.jl

```diff
@@ -0,0 +1,34 @@
+include("../common.jl")
+
+name = "ParU"
+version = v"0.1.3"
+SS_version_str = "7.7.0"
+SS_version = VersionNumber(SS_version_str)
+
+sources = suitesparse_sources(SS_version)
+
+# Bash recipe for building across all platforms
+script = raw"""
+PROJECTS_TO_BUILD="paru"
+CMAKE_OPTIONS+=(
+        -DSUITESPARSE_USE_SYSTEM_SUITESPARSE_CONFIG=ON
+        -DSUITESPARSE_USE_SYSTEM_AMD=ON
+        -DSUITESPARSE_USE_SYSTEM_COLAMD=ON
+        -DSUITESPARSE_USE_SYSTEM_CAMD=ON
+        -DSUITESPARSE_USE_SYSTEM_CCOLAMD=ON
+        -DSUITESPARSE_USE_SYSTEM_UMFPACK=ON
+        -DSUITESPARSE_USE_SYSTEM_CHOLMOD=ON
+    )
+""" * build_script(true)
+
+# Add dependency on SuiteSparse_jll
+dependencies = append!(dependencies, [
+    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"); platforms=filter(!Sys.isbsd, platforms)),
+    Dependency(PackageSpec(name="LLVMOpenMP_jll", uuid="1d63c593-3942-5779-bab2-d838dc0a180e"); platforms=filter(Sys.isbsd, platforms)),
```

Comment:
Is this for a package which is eventually going to be a Julia dep?  `LLVMOpenMP_jll` isn't currently a stdlib, so we can't nilly willy add it as a dep.

Reply:
No. CXSparse, ParU, SPEX, GraphBLAS, and LAGraph will remain outside of stdlibs. The plan is not to add any further sparse deps to the stdlib, with most development happening in packages.